module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Find-EntraPermission.md

```diff
@@ -32,6 +32,7 @@ Find-EntraPermission
  [-ExactMatch]
  [-PermissionType <String>]
  [-Online]
+ [-All]
```

Comment:
The entire syntax section should be this

## SYNTAX

### Search
```
Find-EntraPermission
 [-SearchString] <String>
 [-ExactMatch]
 [-PermissionType <String>]
 [-Online]
 [<CommonParameters>]
```

### All
```
Find-EntraPermission
 [-PermissionType <String>]
 [-Online]
 [-All]
 [<CommonParameters>]
```